src/components/metadata/src/_metadata.scss

```diff
@@ -0,0 +1,34 @@
+.metadata {
+  dt {
+    font-weight: $font-weight-bold;
+
+    @include mq(xs, m) {
+      margin-bottom: 0;
+    }
+
+    &:not(:first-child) {
+      @include mq(xs, m) {
+        margin-top: 0.5rem;
+      }
+    }
+  }
+
+  dd {
+    margin-left: 0; /* As normalize adds a 40px left margin */
+
+    &:not(:last-child) {
+      @include mq(xs, m) {
+        margin-bottom: 0;
```

Comment:
I don't think this is doing anything and so probably the `&:not(:last-child) {...}` can be removed. I think the `margin-top` on the `dt` is handling the spacing between sets.